src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -59,6 +59,13 @@ sub make_figure_of_merit{
             my $color = $c->{"GS_BV$i"};
             my $hw = $c->{"HALFW$i"};
             my $star_prob = _acq_success_prob($date, $t_ccd, $mag, $color, $spoiler, $hw);
+            if (grep(/Bad Acquisition Star/i, @warnings)){
+                my @w = grep(/Bad Acquisition Star/i, @warnings);
+                my ($failed, $total) = parse_bad_acq_warning($w[0]);
```

Comment:
By using the pre-existing Bad Acquisition warnings, the probabilities will be overridden only if a warning is popping up.  The warnings right now are for bad fraction > 0.3 and more than 3 observations.